provider/datagen/Cargo.toml

```diff
@@ -70,7 +70,7 @@ syn = {version = "2", features = ["parsing"], optional = true }
 
 # Other external dependencies
 displaydoc = { version = "0.2.3", default-features = false }
-elsa = "1.7"
+elsa = { git = "https://github.com/Manishearth/elsa", rev = "56d05375eea36596432a0843721d26edf3b0ec75" }
```

Comment:
block?

Reply:
We should change it back to a crates.io version before the 1.3 release. @Manishearth has some things to check up on before pushing another release so he suggested using a git tag in the interim.

Reply:
Is this tracked somewhere as 1.3 blocking?

Reply:
Actually you don't need this anymore

Reply:
I still use it in `test_fallback_options`

Reply:
The git hash and `elsa` release block 1.3 but they shouldn't need to block landing this PR. We've landed git hash dependencies before and we just resolve them near the release. Manish has stated an intention to release `elsa` soon.

Reply:
Previous git hashes have always been diplomat iirc, which is explicitly mentioned in the release checklist. This should at least be tracked somewhere.

Reply:
We also had one for `regex_automata` for awhile.

Issue filed: https://github.com/unicode-org/icu4x/issues/3831